src/context.rs

```diff
@@ -3987,7 +3987,7 @@ impl<'a> ContextWriter<'a> {
 
     if eob == 0 {
       self.bc.set_coeff_context(plane, bo, tx_size, xdec, ydec, 0);
-      return false;
```

Comment:
With this change you aren't setting the coefficient context anymore. This *might* be OK, *if* you verify that the initial value of the coefficient context in bc is correct, *and* it's always restored to that after an RDO rollback.